### DISCUSSION TITLE
对作者表示感谢并建议开启请咖啡功能

### DISCUSSION THREAD

**Discussion by forfudan** [imfuxiao/Hamster#74] [General]
iOS 上的开源输入法又多了一个选择，这是值得鼓励和支持的。在此对作者表达诚挚感谢。并且建议作者在 Readme.md 中放出请咖啡链接。

**Comment by imfuxiao** [comment]
首先非常感谢您的认可,  这是对我最大的鼓励, 也是我继续开发下去的动力.

链接的问题之前考虑过, 但是如您所见, App现在还不完善, 对一个软件工程师来说, 这还远没达到心里的那个目标. 所以就没有放. 

在等等, 等我觉得这个App能达到心里的目标了, 我在放链接. 

谢谢您, 谢谢您的认可.

**Reply by forfudan** [reply]
好的，期待这一天早日到来。

另外我注意到有个专门的schema的仓库。想请问一下，我自己开发的输入方案（形码）是否可以集成到应用中，这样方便用户下载使用？

**Reply by imfuxiao** [reply]
可以的, 但测试是个问题, 建议先将方案自行导入, 测试后没有问题, 在PR合并到 [HamsterInputSchemas](https://github.com/imfuxiao/HamsterInputSchemas) 仓库中.

**Reply by forfudan** [reply]
明白了，我这边 fork 之后导入，Action 成功运行。手机app上导入后也可以使用。是否表明没有问题，可以合并到主 repo 中？

**Reply by imfuxiao** [reply]
是的, 没有问题~
